package.json

```
@@ -23,6 +23,7 @@
     "lint:docs:rule-examples": "node Makefile.js checkRuleExamples",
     "lint:fix": "node Makefile.js lint -- fix",
     "lint:fix:docs:js": "node Makefile.js lintDocsJS -- fix",
+    "lint:knip": "knip",
```

Better description for those who are unfamiliar.

```suggestion
    "lint:imports": "knip",
```